example/spec/testAPI/ruleSpec.js

```diff
@@ -59,31 +47,87 @@ function removeRuleAddedParams(rule) {
  *
  * @returns {undefined} - none
  */
-async function waitForTestExecution() {
+async function waitForTestExecution({
```

Comment:
I pulled this out in this PR into the integration test package: https://github.com/nasa/cumulus/pull/672/files and made it more generic, so if that goes through we'll have to merge but it's not approved yet.

I took your code from here and used it as the base of the function in the integration test package.